lib/erl_type.ml

```diff
@@ -137,6 +138,13 @@ let rec of_etf = function
            result_map_m ~f:E.atom_of_etf elems >>= fun atoms ->
            Ok (Atom atoms)
         end
+     | BinaryTag ->
+        E.string_of_etf elements >>= fun elems_s ->
```

Comment:
It may be a list if `unit` or `base` is greater than 255, right?
If so, could you please add a TODO comment?

http://erlang.org/doc/apps/erts/erl_ext_dist.html#string_ext 
> String does not have a corresponding Erlang representation, but is an optimization for sending lists of bytes (integer in the range 0-255) more efficiently over the distribution

Reply:
thx.
I Improved Etf_util.list_of_etf to assume optimization of ETF.